src/ars/adapters/outbound/http.py

```diff
@@ -33,17 +34,17 @@
 class ClaimValidity(BaseModel):
     """Start and end dates for validating claims."""
 
-    class Config:
-        extra = "forbid"
-        frozen = True
+    model_config = ConfigDict(extra="forbid", frozen=True)
 
     valid_from: DateTimeUTC = Field(
-        ..., description="Start date of validity", example="2023-01-01T00:00:00Z"
+        ..., description="Start date of validity", examples=["2023-01-01T00:00:00Z"]
     )
     valid_until: DateTimeUTC = Field(
-        ..., description="End date of validity", example="2023-12-31T23:59:59Z"
+        ..., description="End date of validity", examples=["2023-12-31T23:59:59Z"]
     )
 
+    # TODO[pydantic]: We couldn't refactor the `validator`, please replace it by `field_validator` manually.
+    # Check https://docs.pydantic.dev/dev-v2/migration/#changes-to-validators for more information.
```

Comment:
Missed something here the bump-pydantic script did not know how to handle.
Interesting, that the TODO was not caught...

Reply:
Oh, didn't realize that, thanks

Reply:
Fixed now.